File: 敏感词过滤/WordFilterHelper_OC/WordFilterHelper_OC/ViewController.m
//
//  ViewController.m
//  WordFilterHelper_OC
//
//  Created by zhangcong on 2017/10/13.
//  Copyright © 2017年 zhangcong. All rights reserved.
//

#import "ViewController.h"
#import "WordFilterHelper.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
    
    NSLog(@"我爱胡锦涛 过滤后 ： %@",[WordFilterHelper.shared filter:@"我爱胡锦涛"]);
}


- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}


@end

File: 敏感词过滤/WordFilterHelper_swift/WordFilterHelper/ViewController.swift
//
//  ViewController.swift
//  WordFilterHelper
//
//  Created by zhangcong on 2017/10/13.
//  Copyright © 2017年 zhangcong. All rights reserved.
//

import UIKit


class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
        
        let str = WordFilterHelper.shareInstance.filter(str: "我爱胡锦涛")
        print(str)
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


}

